api/v3/examples/Setting/GetFields.php

```diff
@@ -240,6 +240,21 @@ function setting_getfields_expectedresult() {
         'description' => '',
         'help_text' => '',
       ),
+      'update_contribution_on_membership_type_change' => array(
+        'group_name' => 'Contribute Preferences',
+        'group' => 'contribute',
+        'name' => 'update_contribution_on_membership_type_change',
+        'type' => 'Integer',
+        'html_type' => 'checkbox',
+        'quick_form_type' => 'Element',
+        'default' => 0,
+        'add' => '4.7',
+        'title' => 'Automatically update related contributions when Membership Type is changed',
+        'is_domain' => 1,
+        'is_contact' => 0,
+        'description' => 'Enabling this setting will updated related contribution of membership(s) excluding autorenewal.',
```

Comment:
Change to: 'Enabling this setting will updated relate contribution of membership(s) except if the membership is paid for with a recurring contribution.'

Note: Autorenewal is not same as recurring contribution. Some memberships that don't autorenew may be paid for by recurring contribution. 

Question: do any payment processors currently support changes to the amount of recurring contributions? If so, we should support updating them.